src/reuse/_util.py

```diff
@@ -211,13 +211,19 @@ def filter_ignore_block(text: str) -> str:
     REUSE_IGNORE_END to remove lines that should not be treated as copyright and
     licensing information.
     """
+    reuse_ignore_begin = "REUSE_IGNORE_BEGIN"
+    reuse_ignore_end = "REUSE_IGNORE_END"
```

Comment:
These should probably be global variables :)

Reply:
@carmenbianca Yeah, I was wondering about that too.  Feel free to put them in an appropriate place.

Reply:
Done.